js/index.js

```diff
@@ -3,29 +3,47 @@
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity input').value;
+  const subt = price.innerHTML * quantity
+  const subtotalPrice = product.querySelector('.subtotal span');
+  subtotalPrice.innerHTML = `${subt}`
+  
+  return subt;
 }
 
 function calculateAll() {
   // code in the following two lines is added just for testing purposes.
   // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
+  //const singleProduct = document.querySelector('.product');
+  //updateSubtotal(singleProduct);
+  
   // end of test
 
   // ITERATION 2
-  //... your code goes here
+  const singleProduct = document.getElementsByClassName('product');
+  let sum = 0;
+  for (let i = 0; i < singleProduct.length; i++){
+    sum += updateSubtotal(singleProduct[i]);
+  }
 
   // ITERATION 3
-  //... your code goes here
+  const totalPrice = document.querySelector("h2 span");
+
+  totalPrice.innerHTML = `${sum}`
```

Comment:
```suggestion
  totalPrice.innerText= `${sum}`
```
Lo mismo que antes si te lo puedes evitar mejor!